.github/workflows/main.yml

```diff
@@ -51,7 +51,7 @@ jobs:
         uses: ./
         with:
           command: cfn-lint -t ./examples/template.yml
-          version: "0.71.1"
+          version: "0.72.0"
```

Comment:
```suggestion
```
Probably don't need to specify a version here, right?

Reply:
Correct 👍